eodag/resources/providers.yml

```
@@ -5928,3 +5928,43 @@
         <<: *day_month_year
     GENERIC_PRODUCT_TYPE:
       dataset: '{productType}'
+---
+!provider # MARK: fedeo_ceda
+  name: fedeo_ceda
+  priority: 0
+  description: CEDA Federation of Earth Science Data Centres via FedEO STAC Catalog
+  roles:
+    - host
+  url: https://fedeo.ceos.org/
+  search: !plugin
+    type: StacSearch
+    api_endpoint: 'https://fedeo.ceos.org/search'
+    need_auth: false
+    ssl_verify: true
+    timeout: 20
+    discover_product_types:
+      fetch_url: 'https://fedeo.ceos.org/series/eo:organisationName/CEDA/'
+      result_type: json
+      results_entry: '$.links[?(@.rel=="child")]'
+      generic_product_type_id: '{$.href#replace_str(r".*/", "")}'
+      generic_product_type_parsable_properties:
+        productType: '{$.href#replace_str(r".*/", "")}'
+        title: '$.title'
+      single_collection_fetch_url: 'https://fedeo.ceos.org/series/eo:organisationName/CEDA/{productType}'
+      single_product_type_parsable_metadata:
+        ID: '{$.assets.enclosure.href#create_name}'
+        abstract: '$.description'
+        instrument: '{$.summaries.instruments#csv_list}'
+        keywords: '{$.keywords#csv_list}'
+        license: '$.license'
+        platform: '{$.summaries.platform#csv_list}'
+        title: '$.title'
+    metadata_mapping:
+      title: '$.properties.title'
```

```suggestion
      title: '{$.properties.title#remove_extension}'
```
title is used to create download directory, this is better without extension